src/main/webapp/js/components/vaccinee.jsx

```diff
@@ -1,14 +1,61 @@
-import React from 'react';
-
-const Vaccinee = () => (
-  <div className="container-fluid">
-    <h1>
-      Vacccine Vacccine Vacccine Vacccine
-      <br />
-      Vacccine Vacccine Vacccine Vacccine
-      <br />
-    </h1>
-  </div>
-);
-
-export default Vaccinee;
+import React, { Component } from 'react';
+import ReactTable from 'react-table';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import 'react-table/react-table.css';
+
+import fetchVaccinees from '../actions/index';
+
+class Vaccinee extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ loading: true });
+    this.props.fetchVaccinees() //eslint-disable-line
+      .then(() => this.setState({ loading: false }));
+  }
+
+  static getTableColumns() {
```

Comment:
you will need to write some utils method to create table columns from field config

Reply:
Should I do it now?

Reply:
I think it can be added later